pkg/storage/unified/search/dashboard.go

```diff
@@ -150,6 +145,9 @@ func (s *DashboardDocumentBuilder) BuildDocument(ctx context.Context, key *resou
 		return nil, err
 	}
 
+	// metadata name is the dashboard uid
+	summary.UID = obj.GetName()
```

Comment:
@ryantxu this is supposed to be `meta.name` right? Doesn't get parsed above in `ReadDashboard()` so setting it here.

Reply:
No harm -- but is this necessary?